bin/api-compare.php

```diff
@@ -31,7 +39,44 @@
             new GetVersionCollectionFromGitRepository(),
             new PickLastMinorVersionFromCollection(),
             new Comparator(
-                new Comparator\BackwardsCompatibility\ClassBased\PropertyRemoved()
+                new ClassBased\MultiClassBased(
```

Comment:
Could probably need documentation (in `docs/` folder) explaining each of this; it's fine to put in a new issue for now

Reply:
Would probably just document each class, no?

If they are unclear, then I suggest correcting the naming instead